feature_list/feature_list.cpp

```diff
@@ -83,6 +84,25 @@ string GetReadableType(FeatureType const & f)
   return result == 0 ? string() : classif().GetReadableObjectName(result);
 }
 
+string GetWheelchairType(FeatureType const & f)
```

Comment:
What is the point? Feature_list is a standalone program, not a library. This is one of the functions related to its operation. Should I put all the functions inside an anonymous namespace? Is that a commonly-used practice?